codewell-fe/src/Components/Comment/Add.js

```diff
@@ -10,6 +10,10 @@ function AddComment(props) {
   const [comment, setComment] = useState();
   const [onCommentAddSuccess, setOnCommentAddSuccess] = useState(0);
 
+  const buttonStyle = {
+    float: 'right'
+  }
```

Comment:
```suggestion
  };
```
let's be consistents